### ISSUE TITLE
[BUG] Filtering HPAs on the new  indexed field returns error

### ISSUE THREAD

**Issue by richard-cox** [rancher/rancher#48479]
**Rancher Server Setup**
- Rancher version: 

**Describe the bug**
- Requests to `/v1/autoscaling.horizontalpodautoscalers?filter=spec.minReplicas=1` fail with `column is invalid [spec.minReplicas]: supplied column is invalid` using `ghcr.io/moio/rancher:2.11.0-qa-xmas.1` which should contain https://github.com/rancher/steve/blob/main/pkg/stores/sqlproxy/proxy_store.go#L93-L98
- Could not see field as column in _fields table in db. also affects other HPA fields

**To Reproduce**
- bring up rancher in container with image from above
- set up & log in
- make http request using above url

**Result**
- error from above

**Expected Result**
- 200 with no resources in response

**Comment by richard-cox** [comment]
Makes sense! Have added the required update to https://github.com/rancher/rancher/issues/48473

**Comment by moio** [comment]
Should be fixed now. Closing in the interest of time, I assume it's been tested at least once
